views/page-widgets/suggested-tasks.php

```diff
@@ -31,9 +31,10 @@
 
 <hr>
 
-<h2 class="prpl-widget-title">
-	<?php \esc_html_e( 'Ravi\'s recommendations', 'progress-planner' ); ?>
-</h2>
+<div class="prpl-dashboard-widget-suggested-tasks">
+	<h2 class="prpl-widget-title">
+		<?php \esc_html_e( 'Ravi\'s recommendations', 'progress-planner' ); ?>
+	</h2>
```

Comment:
@ilicfilip Was this change done by mistake?

Reply:
I think I added wrapper so I can target that h2 specifically, but closing </div> was removed by some merge probably.

Reply:
Yeah, it was removed here: https://github.com/Emilia-Capital/progress-planner/pull/125/commits/167bfba14be6bf561294b994c32064447f3b078d

Reply:
🤔 Why can't we use `.prpl-widget-wrapper.prpl-suggested-tasks` as a selector?